src/multiselect_controls.js

```diff
@@ -293,76 +314,107 @@ export class MultiselectControls {
   }
 
   /**
-   * Maintain the selected blocks set list when updating.
-   * @param {!Blockly.BlockSvg} block The block to update.
+   * Maintain the selected draggable set list when updating.
+   * NOTE: THIS ONLY RUNS FOR BLOCKS UNTIL MORE DRAGGABLES LIKE
+   * WORKSPACE COMMENTS ARE IMPLEMENTED
+   * @param {!Blockly.IDraggable} draggable The draggable to update.
    * @private
    */
-  updateBlocks_(block) {
-    if (block &&
-        (block.isDeletable() || block.isMovable())) {
-      if (this.blockSelection.has(block.id)) {
-        this.blockSelection.delete(block.id);
-        this.justUnselectedBlock_ = block;
-        block.pathObject.updateSelected(false);
+  updateDraggables_(draggable) {
+    if (draggable) {
```

Comment:
This is now hard to read (to many nested if branch). Let's use early return here:
```
if (!draggable) { return; }